debian/copyright

```diff
@@ -0,0 +1,31 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: hdbpp-cm
+Source: https://github.com/tango-controls-hdbpp/hdbpp-es
+
+Files: *
+Copyright: 2014-2017 Elettra - Sincrotrone Trieste S.C.p.A.
+                     Strada Statale 14 - km 163,5 in AREA Science Park
+                     34149 Basovizza, Trieste, Italy.
+License: LGPL-3.0+
```

Comment:
We would need to change this as well (and below) if we agree to use GPL instead of LGPL for the device servers.

Reply:
Updated